doc/time_variance.md

```diff
@@ -1,9 +1,9 @@
 # Time Variance
 
 For the purpose of auditing and analytics modern business information systems
-need to be [time variant](https://en.wikipedia.org/wiki/Time_variance) which
-means they should have the ability to store, track and query data entities that
-change over time. As a [temporal database](https://en.wikipedia.org/wiki/Temporal_database)
+need to be [time variant](https://en.wikipedia.org/wiki/Time_variance) (no wiki entry anymore?). This
```

Comment:
I guess, we need a different link here. Do you have a suggestion @kordano ?

Reply:
We could leave the link out.

Reply:
Ok, I remove it then